### PR TITLE
[RC1] List CNTT mandatory testcases for compliance/certification

### PR DESCRIPTION
**Pull request by collivier**
It lists the testcases already executed in NFVI Compliance
https://lists.opnfv.org/g/opnfv-tsc/message/5717

It also maps them to the requirements being written in
[RC1] Chapter 4.

Then it's linked to the next PRs:
- https://github.com/cntt-n/CNTT/pull/831
- https://github.com/cntt-n/CNTT/pull/833
- https://github.com/cntt-n/CNTT/pull/835
- https://github.com/cntt-n/CNTT/pull/837
- https://github.com/cntt-n/CNTT/pull/839
- https://github.com/cntt-n/CNTT/pull/844
- https://github.com/cntt-n/CNTT/pull/845
- https://github.com/cntt-n/CNTT/pull/848
- https://github.com/cntt-n/CNTT/pull/853

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Comment by collivier** [comment]
@rabi-abdel  @rabi-abdel Are you aware about this labeler issue? It seems not working on forks (all my PRs are failing) https://github.com/actions/labeler/issues/12

**Comment by rabiabdel** [comment]
> @rabi-abdel @rabi-abdel Are you aware about this labeler issue? It seems not working on forks (all my PRs are failing) [actions/labeler#12](https://github.com/actions/labeler/issues/12)

@collivier yes, it doesn't work for forks (not sure how to fix it) but this will not block merge.

File: doc/ref_cert/lfn/chapters/chapter09.md [hunk 1]
```diff
@@ -19,11 +19,47 @@ Define, and describe the purpose of this chapter to be:
 <a name="9.2"></a>
 ## 9.2 RM/RA-1 Requirements
 
-- List out reference to Test Cases Requirements (coming from Chapter 3) and map them to real project test scripts.
+According to [RC1 Chapter04](https://github.com/cntt-n/CNTT/blob/master/doc/ref_cert/lfn/chapters/chapter04.md)
```
Comment (deebhatia) [inline doc/ref_cert/lfn/chapters/chapter09.md]: Use relative links. This will redirect wrongly when compiled with jekyll on `https://cntt-n.github.io/`

Comment (collivier) [reply doc/ref_cert/lfn/chapters/chapter09.md]: https://cntt-n.github.io/ returns 404.
Not a big deal to switch to relative path if it's a global rule here. I'm diving into it.

Comment (deebhatia) [reply doc/ref_cert/lfn/chapters/chapter09.md]: Absolute doesn't work in both. If a person viewing the md files, it breaks.

**Pull request merged**